README.md

```diff
@@ -1,6 +1,35 @@
 # Nukta
 > الجزايريين زهوانيين يحبو يضحكو
 
+## Getting started
+Install:  
+`yarn add nukta.dz` or `npm i --save nukta.dz`
+
+Start using:  
+```js
+const nukta = require('nukta.dz')
+
+console.log(`We have ${nukta.count} jokes available for you! ashba3 da7k ;P`)
+
+// Set all jokes
+console.log(nukta.nukat)
+
+// Find all jokes using a regular expression
+console.log(nukta.search('sellal'))
```

Comment:
Donno . Finst thing that popped in mind lol
Come on .. give me something else :D